File: src/main/java/io/github/persiancalendar/praytimes/AsrMethod.java
package io.github.persiancalendar.praytimes;

// Asr Juristic Methods
public enum AsrMethod {
    Standard(1d), // Shafi`i, Maliki, Ja`fari, Hanbali
    Hanafi(2d); // Hanafi

    // asr shadow factor
    final double asrFactor;

    AsrMethod(double factor) {
        asrFactor = factor;
    }
}

File: src/main/java/io/github/persiancalendar/praytimes/AsrMethod.kt
package io.github.persiancalendar.praytimes

// Asr Juristic Methods
enum class AsrMethod(  // Hanafi
    // asr shadow factor
    val asrFactor: Double
) {
    Standard(1.0),  // Shafi`i, Maliki, Ja`fari, Hanbali
    Hanafi(2.0);
}